.circleci/config.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - run:
           name: Test
           command:
-            dotnet test -c Release
+            dotnet test -c Release --filter Transactions!=Stream
```

Comment:
What about organising those under a trait named feature? I think it might be easier to type and read when we add more filters. Example: `--filter Feature!=StreamTransaction`

Currently we would need to remember all the different trait names, but by using a common trait name like `Feature` we would just need to know about the trait values.